Source/StrategyGame/Public/Player/StrategyPlayerController.h

```diff
@@ -102,7 +102,7 @@ class AStrategyPlayerController : public APlayerController, public IStrategyTeam
 	 * @param	
 	 * @param	
 	 */
-	virtual void GetAudioListenerPosition(FVector& Location, FVector& FrontDir, FVector& RightDir) override;

```

Comment:
That override keyword shouldn't have been removed. It's not important, as you may know, but if the parent class has the same method, it's a good practice to have the override key defined, that way the compiler will check that you are really overriding that method (which exists on `APlayerController).

Reply:
Don't worry btw, I'm gonna change it while merging it.

Reply:
Wait, what? Obviously you changed it because the compiler is not properly compiling it when setting it to override, but the method exists in PlayerController, right? What amb I missing? 😂 

![imatge](https://user-images.githubusercontent.com/153305/82750626-ac143880-9db1-11ea-88aa-e7f910eb2f92.png)